packages/inventory-compliance/src/SystemRulesTable.test.js

```diff
@@ -0,0 +1,157 @@
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+import toJson from 'enzyme-to-json';
+import SystemRulesTable from './SystemRulesTable';
+import { IntlProvider } from 'react-intl';
+jest.mock('@redhat-cloud-services/frontend-components-remediations/remediationsApi');
+import * as remediationsApi from '@redhat-cloud-services/frontend-components-remediations/remediationsApi';
+import { SortByDirection } from '@patternfly/react-table';
+import { TITLE_COLUMN } from './Constants';
+
+const windowInsightsSpy = {
+    chrome: {
+        auth: {
+            getUser: () => {
+                return { then: (callback) => callback() };
+            }
+        }
+    }
+};
+
+const remediationsResponse = {
+    'ssg:rhel7|pci-dss|xccdf_org.ssgproject.content_rule_docker_storage_configured': false,
+    'ssg:rhel7|pci-dss|xccdf_org.ssgproject.content_rule_service_docker_enabled': false
+};
+
+const system = {
+    id: 'aa9c4497-5707-4233-9e9b-1fded5423ef3',
+    name: '3.example.com',
+    profiles: {},
+    __typename: 'System'
+};
+
+const profileRules = [
```

Comment:
Perhaps moving this to a JS with export file under `data/` folder might be better. So the test files are just for testing and we can change data much easily.

Reply:
Sounds good